### DISCUSSION TITLE
Status of Lightspark support page

### DISCUSSION THREAD

**Discussion by eiglow** [lightspark/lightspark#602] [General]
We'll use this thread for discussing the [Status of Lightspark support](https://github.com/lightspark/lightspark/wiki/Status-of-Lightspark-support) page.

I'll start with formatting. At least within the Github editor, it seems like formatting is a huge pain. It looks like there has been some spacing out done, but it doesn't look very good on my screen:
![image](https://user-images.githubusercontent.com/16441870/103451791-0433f580-4d18-11eb-838b-f4858d811e8a.png)

Not sure if this is due to DPI scaling differences on my system. I think a site maybe on github pages or something might be better for this kind of list, that way we would have much more control over formatting, and it'd be easier to manage as the list grows. @huwdp

**Comment by huwdp** [comment]
Hi @eiglow , I used a tool to convert from a spreadsheet to markdown and it padded the columns. The tool is https://tabletomarkdown.com/.

I also use this tool (https://stackedit.io/) when possible. The github editor isn't too great due to it's size so I'm happy to use Github pages if you wanted to

**Comment by huwdp** [comment]
On seconds thoughts I think it would be better to use something like Google Docs instead. I have a link here (https://docs.google.com/spreadsheets/d/164grNRLLLePg-MIdMbEQDwTwsl6FocYC_PMI052ThI8/edit?usp=sharing) to a spreadsheet. What do you think?
